support/scripts/mkgrubimg

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+# allow color coded outputs
+NC="\033[0m"
+RED="\033[0;31m"
+
+# help menu
+usage()
+{
+	echo "Usage: $0 [options] [output file]"
+	echo "Creates bootable GRUB images from Unikraft kernel images."
+	echo ""
+	echo "  -h                         Display help and exit"
+	echo "  -f                         Output format (supported: iso)"
+	echo "  -i                         (optional) Path to initrd directory"
+	echo "  -k                         Path to the Unikraft kernel image"
+	exit 1
+}
+
+# process options
+while getopts "hk:i:f:" options; do
```

Comment:
I prefer to keep one style for variable names. If you name them all with capital letters, do this also for options, like:
```bash
while getopts "hk:i:f:" OPT; do
   case "$OPT" in
```